docs/docusaurus/docs/troubleshooting.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_label: 'Troubleshooting'
+title: 'Troubleshooting'
+description: Learn more about the features and benefits of GX Cloud.
+---
+
+Use the information provided here to help you resolve issues with GX OSS or GX Cloud.
+
+### My Expectation Suite is not generating Expectations
+
+This issue is typically caused when an Expectation fails and `discard_failed_expectations` is set to `True`. To resolve this issue, set `discard_failed_expectations` to `False`.
```

Comment:
To resolve this issue, set `discard_failed_expectations` to `False` during `validator.save_expectation_suite(discard_failed_expectations= False)`